marshmallow/utils.py

```diff
@@ -334,16 +334,31 @@ def callable_or_raise(obj):
     return obj
 
 
-def get_func_args(func):
```

Comment:
@sloria how about moving this version specific functions into compat.py instead of utils.py?